models/pathology_nuclei_segmentation_classification/configs/train.json

```diff
@@ -0,0 +1,502 @@
+{
+    "imports": [
+        "$import glob",
+        "$import os",
+        "$import skimage"
+    ],
+    "bundle_root": "/workspace/Code/model-zoo/models/pathology_nuclei_segmentation_classification",
```

Comment:
here can the `bundle_root` be `$os.getcwd()` as @holgerroth suggested in https://github.com/Project-MONAI/model-zoo/issues/247 ? Thanks!

Reply:
Have updated it in https://github.com/Project-MONAI/model-zoo/pull/245/commits/09a2938f73b2efed915f6e141bd262144e85d944, thanks!